primitives/storage/src/lib.rs

```diff
@@ -62,7 +62,10 @@ pub struct StorageChild {
 pub struct Storage {
 	/// Top trie storage data.
 	pub top: StorageMap,
-	/// Children trie storage data by storage key.
+	/// Children trie storage data.
+	/// The key does not including prefix, for the `default`
+	/// trie kind, so this is exclusively for the `ChildType::ParentKeyId`
+	/// tries.
 	pub children: std::collections::HashMap<Vec<u8>, StorageChild>,
```

Comment:
Yes, but I also have a mix feeling calling it 'default', if at some point the default change.
Considering it can be rename if the default change, I would also think it can be rename when a second child type appears. But that is probably not a good idea as it will probably be forgotten, so 'children_default' or 'default' could be fine.
I like 'default' as it is shorter but yes, default_children is probably better.